dev-packages/application-manager/src/generator/backend-generator.ts

```diff
@@ -61,8 +65,8 @@ function start(port, host) {
     });
 }
 
-module.exports = (port, host) => Promise.resolve()${this.compileBackendModuleImports(backendModules)}
-    .then(() => start(port, host)).catch(reason => {
+module.exports = (port, host, argv) => Promise.resolve()${this.compileBackendModuleImports(backendModules)}
```

Comment:
I cannot find any clients using a new arg in this PR and there are other irrelevant changes.

Reply:
found it in another PR: https://github.com/theia-ide/theia/pull/3017/files#diff-53844aaa6866e66a0e9034a821601e2eR199

I think we can keep only https://github.com/theia-ide/theia/pull/3017 and close all other PRs for each commit. Travis has a better job to do than rebuilding the same changes 5 times.

Reply:
This is the only way I found to have it test the individual commits.